ADMINS-FOR-ISTIO.md

```diff
@@ -6,21 +6,21 @@ access.
-| [Google Analytics](https://analytics.google.com/analytics/web/) | Analytics for *istio.io | [jasonall](https://github.com/jasonall)
-| [Google Calendar - Community Events](https://calendar.google.com/calendar/embed?src=i10ogf58krfbrsjai5qi16g4do%40group.calendar.google.com&ctz=America%2FLos_Angeles) | Community event scheduling | [oaktowner](https://github.com/oaktowner)
+| [CircleCi](https://circleci.com) | Secondary CI system | [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
+| [Discourse](https://discuss.istio.io) | Discussion forums | [askmeegs](https://github.com/askmeegs), [linsun](https://github.com/linsun), [craigbox](https://github.com/craigbox)
+| [Docker Hub](https://hub.docker.com) | Container registry | [brian-avery](https://github.com/brian-avery), [costinm](https://github.com/costinm), [mandarjog](https://github.com/mandarjog), [vadimeisenbergibm](https://github.com/vadimeisenbergibm), [frankbu](https://github.com/frankbu)
+| [GitHub](https://github.com/istio) | Source control, bug tracking | [howardjohn](https://github.com/howardjohn), [brian-avery](https://github.com/brian-avery), [linsun](https://github.com/linsun), [louiscryan](https://github.com/louiscryan), [smawson](https://github.com/smawson)
+| [Google Analytics](https://analytics.google.com/analytics/web/) | Analytics for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Google Calendar - Community Events](https://calendar.google.com/calendar/embed?src=i10ogf58krfbrsjai5qi16g4do%40group.calendar.google.com&ctz=America%2FLos_Angeles) | Community event scheduling | [macruzbar ](https://github.com/macruzbar)
 | [Google Calendar - Working Groups](https://calendar.google.com/calendar/embed?src=4uhe8fi8sf1e3tvmvh6vrq2dog%40group.calendar.google.com&ctz=America%2FLos_Angeles) | Working group scheduling | All working group leads
-| [Google Cloud DNS](https://cloud.google.com/dns) | DNS hosting for *istio.io | [jasonall](https://github.com/jasonall)
-| [Google Custom Search Engine](https://cse.google.com/cse/) | Search engine for *istio.io | [jasonall](https://github.com/jasonall), [rshriram](https://github.com/rshriram)
-| [Google Domains](https://domains.google.com) | Domain registration for *istio.io | [jasonall](https://github.com/jasonall)
+| [Google Cloud DNS](https://cloud.google.com/dns) | DNS hosting for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Google Custom Search Engine](https://cse.google.com/cse/) | Search engine for *istio.io | [davidhauck](https://github.com/davidhauck)
+| [Google Domains](https://domains.google.com) | Domain registration for *istio.io | [craigbox](https://github.com/craigbox)
```

Comment:
I don't have admin access to this, but if you can find who does, you can give it to me

Reply:
@davidhauck has made it so